shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
Hey, I don't know what I've done.
I was working on the master branch (instead of the branch I created) and I committed and pushed my code there also, then I tried to delete it (using `git reset --hard <sha1-commit-id>` then `git push origin HEAD --force`) but I don't think that's it's deleted (because it still says (in the master branch) this branch is 1 commit ahead and my commit shows in the "latest commit" also).